stable/aws-load-balancer-controller/values.yaml

```diff
@@ -72,6 +72,9 @@ enableCertManager: false
 # ingresses without ingress class annotation and ingresses of type alb
 ingressClass: alb
 
+# Set to use KIAM or kube2iam for example
```

Comment:
Lets say something like
The AWS region for the kubernetes cluster. Set to use KIAM or kube2iam for example.